### PR TITLE
chore: Crowdin Enterprise Setup

### PR DESCRIPTION
**Pull request by dozortsev**
## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: .github/workflows/crowdin-download.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          localization_branch_name: l10n_${{ steps.vars.outputs.branch_name }}
+          create_pull_request: true
+          pull_request_title: 'New or updated Crowdin translations'
+          pull_request_body: '## Context
+
+            Automated pull request pulling in new or updated translations from Crowdin
+
+            ## Checklist
+
+            - [x] I have considered the impact of this change and added a [Change Classification](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
```
Comment (dozortsev) [inline .github/workflows/crowdin-download.yml:39]:
```suggestion
```

File: .github/workflows/crowdin-download.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          create_pull_request: true
+          pull_request_title: 'New or updated Crowdin translations'
+          pull_request_body: '## Context
+
+            Automated pull request pulling in new or updated translations from Crowdin
+
+            ## Checklist
+
+
+            - [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)'
```
Comment (dozortsev) [inline .github/workflows/crowdin-download.yml:40]:
```suggestion
```

File: .github/workflows/crowdin-download.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        uses: crowdin/github-action@1.3.1
+        with:
+          upload_translations: false
+          download_translations: true
+          localization_branch_name: l10n_${{ steps.vars.outputs.branch_name }}
+          create_pull_request: true
+          pull_request_title: 'New or updated Crowdin translations'
+          pull_request_body: '## Context
+
+            Automated pull request pulling in new or updated translations from Crowdin
```
Comment (dozortsev) [inline .github/workflows/crowdin-download.yml:35]:
```suggestion
            Automated pull request pulling in new or updated translations from Crowdin'
```

File: .github/workflows/crowdin-download.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - cron: '0 7 * * 1-5'
+  workflow_dispatch:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+      - 'dependabot-**'
+
+jobs:
+  download-from-crowdin:
+    runs-on: [self-hosted, production, medium]
```
Comment (matebek-tw) [inline .github/workflows/crowdin-download.yml:14]: I believe public repositories cannot run jobs on self-hosted runners.

Comment (dozortsev) [reply .github/workflows/crowdin-download.yml:14]: I see, let me fix that to `ubuntu-latest` (as in our `ci-cd.yml` config)

File: .github/workflows/crowdin-download.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+          # signing commits
+          gpg_private_key: ${{ secrets.GPG_SIGN_KEY }}
+          github_user_email: 'tw-actions@transferwise.com'
+
+          # integration settings
+          base_url: 'https://transferwise.crowdin.com'
+          crowdin_branch_name: ${{ steps.vars.outputs.branch_name }}
+          token: ${{ secrets.CROWDIN_ENTERPRISE_TOKEN }}
+          project_id: ${{ secrets.CROWDIN_PROJECT_ID }} # Add as repository secret — number from transferwise.crowdin.com project URL
```
Comment (OFFLlNE) [inline .github/workflows/crowdin-download.yml:42]: Might need to add 681 here as well as we do in `crowdin.yml`

File: crowdin.yml [hunk 6]
```diff
@@ -1,4 +1,7 @@
 project_identifier: neptune-web-components
+project_id: 681
+api_token: CROWDIN_ENTERPRISE_TOKEN
```
Comment (OFFLlNE) [inline crowdin.yml:3]: it is possible we might need to make it `api_token_env`. But happy to try with this one first :)

File: .github/workflows/crowdin-download.yml (hunk 5 above)
Comment (dozortsev) [reply .github/workflows/crowdin-download.yml:42]: actually vise versa, I added `CROWDIN_PROJECT_ID` in `crowdin.yml`

File: crowdin.yml (hunk 6 above)
Comment (dozortsev) [reply crowdin.yml:3]: I prefer we use current one, they seem sort of same https://support.crowdin.com/configuration-file/#api-credentials-from-environment-variables

**Review by OFFLlNE — APPROVED** [review]

**Review by matebek-tw — APPROVED** [review]

**Pull request merged**